packages/ketcher-core/src/application/editor/actions/sgroup.ts

```diff
@@ -300,6 +309,44 @@ function fromAtomAction(restruct, newSg, sourceAtoms) {
   );
 }
 
+function fromQuerySGroupAction(restruct, newSg, sourceAtoms, targetAtoms) {
```

Comment:
Would it be possible to have types for these parameters? Also, can we use the full name "newSGroup", so it's less confusing?

Reply:
done